src/librbd/internal.cc

```diff
@@ -1351,9 +1351,7 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
     int r = ictx->state->open(true);
     if (r < 0) {
       ldout(cct, 2) << "error opening image: " << cpp_strerror(-r) << dendl;
-      if (r != -ENOENT) {
```

Comment:
@dillaman do you mean on -ENOENT i have to check if we can get the image id from `RBD_DIRECTOY` by image name, and then try to open it again?